node/pkg/lifecycle/startup/startup.go

```diff
@@ -890,16 +893,17 @@ func createIPPool(ctx context.Context, client client.Interface, cidr *cnet.IPNet
 			Name: poolName,
 		},
 		Spec: api.IPPoolSpec{
-			CIDR:         cidr.String(),
-			NATOutgoing:  isNATOutgoingEnabled,
-			IPIPMode:     ipipMode,
-			VXLANMode:    vxlanMode,
-			BlockSize:    blockSize,
-			NodeSelector: nodeSelector,
+			CIDR:             cidr.String(),
+			NATOutgoing:      isNATOutgoingEnabled,
+			IPIPMode:         ipipMode,
+			VXLANMode:        vxlanMode,
+			BlockSize:        blockSize,
+			NodeSelector:     nodeSelector,
+			DisableBGPExport: bgpExportDisabled,
 		},
 	}
 
-	log.Infof("Ensure default IPv%d pool is created. IPIP mode: %s, VXLAN mode: %s", version, ipipMode, vxlanMode)
+	log.Infof("Ensure default IPv%d pool is created. IPIP mode: %s, VXLAN mode: %s, DisableBGPExport: %v", version, ipipMode, vxlanMode, bgpExportDisabled)
```

Comment:
Shall we replace `%v` with `%t` since we know it is a boolean variable?

Reply:
Fixed with [94d2ae7](https://github.com/projectcalico/calico/pull/6391/commits/94d2ae7bfd0c663f8cadadb39cc403f3d0fc0168)